content/docs/services/cloud-gov-service-account.md

```diff
@@ -16,7 +16,7 @@ To set up your application to be deployed with an automated system, you need a d
 
 Plan Name | Description | Price
 --------- | ----------- | -----
-`space-deployer` | Provision one account with "SpaceDeveloper" access to the space where the service is created | Free
+`space-deployer` | An account for continuous deployment, limited to a single space | Free
```

Comment:
Suggest "A service account for continuous deployment, limited to a single space" to emphasize that this is a service account instead of a human account